### PR TITLE
Standardisation of predefined types for system fittings issue 142

### PR DESCRIPTION
**Pull request by SergejMuhic**
1. Harmonized PDTs in question
2. Did not do fitting psets (I guess there was some merging of psets and properties)

### CODE REVIEW

**Review by pjanck — APPROVED** [review]

File: IFC4x3/Constants/r/REDUCER_0OPZShjzb5DPiavKDNXHJp.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="REDUCER_0OPZShjzb5DPiavKDNXHJp" Name="REDUCER" UniqueId="1866372b-b7d9-4535-9b24-e543578514f3">
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="REDUCER" UniqueId="1866372b-b7d9-4535-9b24-e543578514f3">
```
Comment (larswik) [inline IFC4x3/Constants/r/REDUCER_0OPZShjzb5DPiavKDNXHJp.xml]: Id-attribute missing?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml [hunk 2]
```diff
@@ -6,9 +6,11 @@
 	</Localization>
 	<Constants>
 		<DocConstant xsi:nil="true" href="BEND_22McF8Isf91hv1nF_4I56O" />
+		<DocConstant xsi:nil="true" href="CONNECTOR_3a4VU6NkD9DPvso_Qb71av" />
 		<DocConstant xsi:nil="true" href="CROSS_3hKhHGisjAD8PqTWi40abu" />
-		<DocConstant xsi:nil="true" href="REDUCER_0OPZShjzb5DPiavKDNXHJp" />
+		<DocConstant xsi:nil="true" href="JUNCTION_0rhNC_wFT94xjdqK4E1gd" />
```
Comment (larswik) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml:12]: Intentional removal of REDUCER?

Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml:12]: Yes, the issues @aothms gave us had it removed but I see now in the documentation that it is merely deprecated. We need some clarification I guess. Good thing we have scheduled a meeting for next week.

Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml:12]: Apparently, this was reverted.

Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml:12]: I have reverted also, see [7ba010c](https://github.com/bSI-InfraRoom/IFC-Specification/pull/420/commits/7ba010cecb40e2f42d347ba5ca65a2a106a325df)

Comment (aothms) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/Types/IfcCableCarrierFittingTypeEnum/DocEnumeration.xml:12]: I'm a bit behind on updating you on the changes that came out of the compatibility assessment

**Review by pjanck — APPROVED** [review]
lftm

**Comment by SergejMuhic** [comment]
@larswik @anborr  fine by you? need one more green

**Review by larswik — APPROVED** [review]

**Pull request merged**